frontend/packages/dev-console/src/components/project-access/project-access-form-utils.ts

```diff
@@ -5,7 +5,22 @@ import {
   ProjectAccessRoles,
 } from './project-access-form-utils-types';
 
-export const filterRoleBindings = (roleBindings: RoleBinding[], roles): RoleBinding[] => {
+export const defaultAccessRoles = {
+  admin: 'Admin',
+  edit: 'Edit',
+  view: 'View',
+};
+
+export type Roles = {
+  [key: string]: string;
+};
+
+export const getAvailableAccessRoles = () => {
+  if (!window.SERVER_FLAGS.projectAccessClusterRoles) return undefined;
+  return JSON.parse(window.SERVER_FLAGS.projectAccessClusterRoles);
+};
+
+export const filterRoleBindings = (roleBindings: RoleBinding[], roles: Roles): RoleBinding[] => {
   return _.filter(roleBindings, (user: RoleBinding) => _.keys(roles).includes(user.roleRef.name));
 };
```

Comment:
Also a personal preference which you can also ignore.. The Roles type is a "frontend type" which is not required for this utils function. I would simplify this and change the signature to `clusterRoleNames: string[]` and extract this data if required in the calling component.

```suggestion
export const filterRoleBindings = (roleBindings: RoleBinding[], clusterRoleNames: string[]): RoleBinding[] => {
  return _.filter(roleBindings, (user: RoleBinding) => clusterRoleNames.includes(user.roleRef.name));
};
```